manim/animation/creation.py

```diff
@@ -361,22 +380,10 @@ def __init__(
             run_time=run_time,
             lag_ratio=lag_ratio,
             rate_func=lambda t: -rate_func(t) + 1,
+            reverse=not reverse,
```

Comment:
I have a very slight preference for setting the default value of the `kwarg` to `True` so that `reverse=reverse` can be passed here.

Semantically, I think it's cleaner if `reverse` acts the same for both `Write` and `Unwrite`.

Reply:
To be honest, I find the `reverse` parameter quite confusing. What about changing it to `left_to_right` where the default value for `Write` is `True` and the default value for `Unwrite` is `False`?

Reply:
-1: `Write` isn't just an animation used for text, and for non-text mobjects `left_to_right` doesn't necessarily make sense.

We can discuss changing the parameter name, but I'd rather do that in a follow-up issue or PR.

Reply:
Good point, I'll follow your suggestion then.